### ISSUE TITLE
TBE client crash when placing Crude Burner

### ISSUE THREAD

**Issue by laceflowermc** [mrh0/createaddition#65]
Client crashes when placing a crude burner in the world.
https://pastebin.com/GVWkv1VQ

Mod version: 20210515b
Forge version: 36.1.0
Create version: 3.1c

**Comment by mrh0** [comment]
I have not been able to reproduce this crash, what other mods are you running?

**Comment by uLLeticaL** [comment]
Same crash here, it works fine in single player, but CTD's when placing the Crude Burner down on a server.

**Versions**
Forge `36.1.17`

Client & Server Mods
```
create-mc1.16.5_v0.3.1c.jar
createaddition-1.16.5-20210515c.jar
jei-1.16.5-7.7.0.93.jar
```

**Comment by mrh0** [comment]
Fixed in 20210517a
